lineman/app/components/membership_requests_page/membership_requests_page_controller.coffee

```diff
@@ -0,0 +1,30 @@
+angular.module('loomioApp').controller 'MembershipRequestsPageController', ($routeParams, $rootScope, Records, LoadingService, AbilityService, FlashService) ->
+  $rootScope.$broadcast('currentComponent', { page: 'membershipRequestsPage'})
+
+  @init = (group) =>
+    @group = group
+    Records.membershipRequests.fetchPendingByGroup(group.key, per: 100)
+    Records.membershipRequests.fetchPreviousByGroup(group.key, per: 100)
+
+  Records.groups.findOrFetchByKey($routeParams.key).then @init, (error) ->
+    $rootScope.$broadcast('pageError', error, group)
+
+  @pendingRequests = =>
+    @group.pendingMembershipRequests()
+
+   @previousRequests = =>
+    @group.previousMembershipRequests()
+
+  @approve = (membershipRequest) =>
+    Records.membershipRequests.approve(membershipRequest).then ->
+      FlashService.success "membership_requests_page.messages.request_approved_success"
+
+
+  @ignore = (membershipRequest) =>
+    Records.membershipRequests.ignore(membershipRequest).then ->
+      FlashService.success "membership_requests_page.messages.request_ignored_success"
+
+  @noPendingRequests = =>
+    @pendingRequests.length == 0
+
```

Comment:
@noPendingRequests function doesn't seem to work - had to take it out of the view...